model/task/task.go

```diff
@@ -155,6 +156,10 @@ type Task struct {
 	GeneratedTasks bool `bson:"generated_tasks,omitempty" json:"generated_tasks,omitempty"`
 	// GeneratedBy, if present, is the ID of the task that generated this task.
 	GeneratedBy string `bson:"generated_by,omitempty" json:"generated_by,omitempty"`
+	// GeneratedJSON is the the configuration information to create new tasks from.
+	GeneratedJSON []json.RawMessage `bson:"generate_json" json:"generate_json"`
```

Comment:
isn't this a byte slice rather than a string?

Reply:
Yes, but I don't understand your question. The type is a `[]json.RawMessage` here.

Reply:
Should we not just make it a slice of strings so that it's easier to read in the DB?